docs/source/packages/files.rst

```diff
@@ -18,3 +18,27 @@ Reference
    :maxdepth: 2
 
    ../classes/zos_files/index
+
+Interacting with z/OS Unix via z/OSMF
+===============================
+
+The Zowe Client Python SDK leverages the z/OS Management Facility (z/OSMF) REST interface to interact with Unix System Services (USS) on z/OS. 
+Rather than connecting directly to USS, the Python SDK/the SDK uses z/OSMF as a standardized conduit for file operations and other USS functionalities. 
+This design offers several benefits:
+
+- **Standardization:** z/OSMF provides a consistent REST API for interacting with various z/OS components, including USS.
+- **Security and maintainability:** By utilizing z/OSMF, the Python SDK benefits from its built-in authentication, logging, and error-handling mechanisms, making integration more secure and manageable.
+- **Simplified integration:** The REST-based approach reduces the complexity of direct USS interactions, allowing for easier maintenance and future enhancements.
+
+In summary, it might appear that all USS functionality is routed through z/OSMF, this approach is intentional, providing a secure and manageable interface to z/OS USS.
```

Comment:
If possible, please use consistent terminology.
I believe `z/OS Unix` is preferred.